setup.py

```diff
@@ -61,6 +64,8 @@
             "tox",
             "tensorflow>=2.6.0",
             "torch>=1.11",
+            "jax==0.4.3",
+            "jaxlib==0.4.3"

```

Comment:
Is the JAX dependency just from [here](https://github.com/lillian542/amazon-braket-pennylane-plugin-python/pull/2/files#diff-d647125628ca86da2d7c9867c269c57e1e1fe0932cb6c162c94c23203aa8eef6R36)? I'd be reluctant to introduce a JAX dependency in the plugin. Can we use `qml.math`?

Reply:
It's to avoid this:

![image](https://user-images.githubusercontent.com/38584660/227269735-e3cc02b5-681c-4229-9102-e3d4eee93151.png)

Reply:
I guess I could make a dummy operator for testing that contains the pulses and settings, which should be all the hardware needs. I *think* that would remove all the jax dependency stuff in the Braket plugin. 

Using this device _is_ jax dependent, because it needs a `ParametrizedEvolution`, but I guess that can be something the Braket plugin doesn't touch and that we just enforce on the PL side.

Reply:
But that does mean if we change ParametrizedEvolution in a way that breaks the plugin, we won't know, because the tests aren't using the real operator, their using some replacement thing that won't be updated when `ParametrizedEvolution` is. 

An alternative would be to try to detangle `ParametrizedEvolution` from `jax` a little, so that it only raises this warning when it does a jax-dependent thing. This would be using PyTrees and calling the matrix, both of which we shouldn't need to do when running it on hardware.

Reply:
> I am worried we might find ourselves in a pitfall of patching no-jax-import tests which have been a pain in the past

This is indeed where I find myself 😵‍💫

Reply:
jax dependencies removed!